### DISCUSSION TITLE
error in codespace for 4-3-python-rasterio

### DISCUSSION THREAD

**Discussion by Alange24** [ua-gist604b-s24/syllabus#24] [General]
Keep getting this error when starting codespace:  the error states "This codespace is currently running in recovery mode due to a configuration error.  Please review the creation logs, update your dev container configuration as needed, and run the rebuild container command to rectify.

I've tried opening codespace a number of times and tried running a Rebuild Container command but don't know if I did it correctly.
Thanks!

**Comment by LenaKemp** [comment]
Hi, did you make sure you chose the appropriate Machine type of 16-core (not the default of 2-core)? If so, the professor also discusses your exact error in the most recent zoom recording from 04/01/2024 beginning around the time 14:39. https://arizona.hosted.panopto.com/Panopto/Pages/Sessions/List.aspx#folderID=%2297ef9efa-336a-4834-9815-b129011280c2%22

If that did not help, luckily there is a zoom meeting tonight if no one else responds. Kind regards.

**Reply by Alange24** [reply]
Thanks Lena, I forgot to mention in my post that the max I have for machine type is 4 core, no 8 or 16 core.
